src/_backend_agg.cpp

```diff
@@ -212,8 +212,8 @@ agg::rect_i RendererAgg::get_content_extents()
 
     r.x1 = std::max(0, r.x1 - 1);
     r.y1 = std::max(0, r.y1 - 1);
-    r.x2 = std::max(r.x2 + 1, (int)width);
-    r.y2 = std::max(r.y2 + 1, (int)height);
+    r.x2 = std::min(r.x2 + 1, (int)width);
+    r.y2 = std::min(r.y2 + 1, (int)height);
```

Comment:
Yeah -- we could take the -1's out and see what happens to the test suite.  Now that we're not doing fuzzy testing, we shouldn't know pretty quickly if it has negative consequences.